### ISSUE TITLE
Home page does not populate articles until lots of scrolling happens

### ISSUE THREAD

**Issue by Mellen** [thepracticaldev/dev.to#4057]
The home page only displays one article, and three "article loading" place holders. The place holders turn into article previews only after I scroll past them (so I miss the first preview).

This is not a case of not waiting long enough. I can leave the page in that state forever.

To reproduce the bug open the home page.

**Expected behavior**

The place holders should load article previews when they come into view (in my case as soon as I hit the home page) and not after I scroll past them.

**Screenshots**
At first:
![View when landing on the home page](https://user-images.githubusercontent.com/127148/65134392-21966080-d9fc-11e9-8c4f-378cc67d9f93.PNG)

Then scroll down a bit:
![View after 4 clonks of the scroll wheel](https://user-images.githubusercontent.com/127148/65134402-24915100-d9fc-11e9-926d-e25ee9840c88.PNG)

**Desktop (please complete the following information):**

- OS: Windows 10
- Browser: Firefox
- Version: 69.0 (64bit)

**Comment by maestromac** [comment]
Thank you for this raising this @Mellen  . 

This looks like duplicate of https://github.com/thepracticaldev/dev.to/issues/1805. Please continue the conversation there.
